### DISCUSSION TITLE
Custom Name Above Head

### DISCUSSION THREAD

**Discussion by thesamgordon** [Minestom/Minestom#1429] [Help] [answered]
Hi,

I was just wondering if there is any packet to set the player's name above their head. 
Things I've tried:

- PlayerInfo.UPDATE_DISPLAY_NAME
- PlayerInfo.ADD_PLAYER
- Player#setDisplayName
- Player#setCustomName

I understand that you can do this with scoreboard teams but it would be much easier to do with packets.

Thanks :)

**Comment by Tofaa2** [comment]
Minestom provides an API for creating and defining scoreboards and their properties, I believe [BelowNameTag](https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/scoreboard/BelowNameTag.java) is what you're looking for. For more examples you can see the [scoreboard](https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/scoreboard/) package

**Comment by emortaldev** [comment, marked as answer]
You can use an armourstand that rides you. I think scoreboard teams can only change the prefix, suffix and name colour to the pre 1.16 colours

**Reply by thesamgordon** [reply]
Thanks! I think I'l go with this as I hate dealing with teams.
